system/doc/reference_manual/documentation.md

```diff
@@ -145,6 +145,9 @@ module and then go into greater details. For example:
 There are three reserved metadata keys for `-moduledoc`:
 
 - `since` - Shows in which version of the application the module was added.
+  If this is added, all functions, types and callbacks within will also receive
```

Comment:
Do we use the Oxford comma `If this is added, all functions, types, and callbacks`?
I think I read somewhere that we do, but I may be mistaken